Tasks/DockerInstallerV0/src/utils.ts

```diff
@@ -61,8 +61,13 @@ function getDockerDownloadURL(version: string, releaseType: string): string {
     switch (os.type()) {
         case 'Linux':
             platform = "linux";
-            if (os.arch() === "arm64") {
-                architecture = "aarch64";
+            switch(os.arch()){
```

Comment:
NIT: use a `.ts` formatter on this file once.

Reply:
@ds-ms Ran VisualStudio Code formatting on the file. Initially I avoided this because the rest of the file is not formatted either. Should be fine now.

Reply:
That's cool :)